apps/common/controllers/add-another-address-loop.js

```diff
@@ -19,14 +19,28 @@ module.exports = class AddAnotherAddressController extends BaseController {
     const hasAddresses = req.sessionModel.get(`${this.field}Addresses`) ? true : false;
     const items = _.map(addresses, (value, key) => ({
       id: key,
-      address: value.address
+      address: value.address,
+      categories: this.translateCategories(req, value.categories)
     }));
     return Object.assign({}, locals, {
       items,
-      hasAddresses
+      hasAddresses,
+      hasCategories: _.sample(addresses).categories !== undefined
     });
   }
 
+  translateCategories(req, values) {
+    if (!Array.isArray(values)) {
+      values = [values];
+    }
+    const categories = values.map(value => {
+      let key = `fields.location-address-category.options.${value}.label`;
+      let result = req.translate(key);
+      return result === key ? value : result;
+    }).join('\n');
+    return categories;
```

Comment:
If `categories` is only used once, you could simply return the result of `values.map`